articles/libraries/lock-android/delegation-api.md

```diff
@@ -0,0 +1,44 @@
+---
+toc_title: Delegation API
+description: Integrate with third-party apps with the delegation API.
+---
+
+# Lock Android: Delegation API
+
+After a successful authentication, you can request credentials to access third party apps like Firebase or AWS that are configured in your Auth0 App's Add-On section. In order to do that you need to make a request to our [Delegation API](/auth-api#!#post--delegation) using a valid JWT.
+
+Here's an example
+```java
+Lock lock = LockContext.getLock(this);
+AuthenticationAPIClient client = lock.getAuthenticationAPIClient();
+String apiType = "firebase";
+String token = .... //Your Auth0 id_token of the logged in User
+Map<String, Object> parameters = ParameterBuilder.newEmptyBuilder()
+        .set("id_token", token)
+        .set("api_type", apiType)
+        .asDictionary();
+client
+    .delegation()
+    .addParameters(parameters).start(new BaseCallback<Map<String, Object>>() {
+        @Override
+        public void onSuccess(Map<String, Object> payload) {
+            //Your Firebase token will be in payload        
```

Comment:
Maybe mention where each thing is in the payload? That way I don't have to inspect it myself